### ISSUE TITLE
Disappearing transactions

### ISSUE THREAD

**Issue by holiman** [ethereum/go-ethereum#14719]
#### System information

Geth version: `1.6.6`

##### Description

Transactions go 'missing', and nonces are reused. 

* happening from one "from-address"
* Seems to occurs more often with increased concurrency (not sure) 
* 2-3 workers with RPC-connections at once against geth, so at least some paralellism involved
* One instance of geth
* `eth_sendTransaction` is used. 

> The last time it happened, I dont believe we were under "heavy load" but i'm not sure what quantify's heavy load.  i would say no more than 100 tx's behind.

Running with `--targetgaslimit 1000000 --cache 1024 --txpool.accountqueue 8192` on the cmd line

Ticket filed on behalf of a user.

**Comment by holiman** [comment]
My theory is that transactions are queued up internally, but after two hours removed from 'local' cache, and thus they become removed due to the general pool cleaning. 

These would by default start cropping the pending/queued to the following sizes: 

``` 
--txpool.accountslots value  Minimum number of executable transaction slots guaranteed per account (default: 16)
--txpool.accountqueue value  Maximum number of non-executable transaction slots permitted per account (default: 64)
```

**Comment by richiela** [comment]
I already up accountqueue to 8192 ... should i try upping accountslots as well?

**Comment by holiman** [comment]
I believe this is fixed with 1.7.0.
